### ISSUE TITLE
Src Skidded from hackermode

### ISSUE THREAD

**Issue by ignSKRRRTT** [Crab-Game-Hacks/EasyInjector#2]
Hey! I'd just like to inform everyone looking at this that none of this code is from the creator. He stole all of it from another cheat called "hackermode" and doesn't credit them at all.

If you would like further proof and him actually admitting that its stolen code please dm me on discord SKRRRTT#9999

**Comment by Crab-Game-Hacks** [comment]
The mod menu is CGHM, the injector & the modded server are not.
I can put this in the read me if its so important.

**Comment by ignSKRRRTT** [comment]
Yeah but the fact you are going around claiming its your own cheat is the issue. No where in the cheat or the discord do you disclaim that the cheat is someone else's. You also feel the need to talk down on other cheats when the cheat you own isn't even yours

**Comment by Crab-Game-Hacks** [comment]
Credit is given in the readme.

**Comment by ignSKRRRTT** [comment]
Yeah look at the commit time dumbass you added it after I called you out LOOOOL 

https://github.com/Crab-Game-Hacks/EasyInjector/commit/ad3d20c5b51c3f48b9eafdc36dcf6ece7278e24f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Comment by Crab-Game-Hacks** [comment]
Either way, its there. Continue to complain if you wish.
